### PR TITLE
K8SPSMDB-951: Fix custom PBM config cr.yaml example

### PR DESCRIPTION
**Pull request by inelpandzic**
[K8SPSMDB-951](https://jira.percona.com/browse/K8SPSMDB-951) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

**CHANGE DESCRIPTION**
---
**Problem:**
Fix custom PBM config cr.yaml example.

**CHECKLIST**
---
**Jira**
- [x] Is the Jira ticket created and referenced properly?
- [x] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [x] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [x] Is an E2E test/test case added for the new feature/change?
- [x] Are unit tests added where appropriate?
- [x] Are OpenShift compare files changed for E2E tests (`compare/*-oc.yml`)?

**Config/Logging/Testability**
- [x] Are all needed new/changed options added to default YAML files?
- [x] Did we add proper logging messages for operator actions?
- [x] Did we ensure compatibility with the previous version or cluster upgrade process?
- [x] Does the change support oldest and newest supported MongoDB version?
- [x] Does the change support oldest and newest supported Kubernetes version?

[K8SPSMDB-951]: https://perconadev.atlassian.net/browse/K8SPSMDB-951?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: deploy/cr.yaml [hunk 1]
```diff
@@ -560,25 +560,26 @@ spec:
...
-#      mongodLocation: /usr/bin/mongo
-#      mongodLocationMap:
-#        "node01:2017": /usr/bin/mongo
-#        "node03:27017": /usr/bin/mongo
+#    configuration:
+#      backupOptions:
+#        priority:
+#          "localhost:28019": 2.5
+#          "localhost:27018": 2.5
+#        compression: gzip
```
Comment (nmarukovich) [inline deploy/cr.yaml:568]: compression should be on the level above

Comment (nmarukovich) [reply deploy/cr.yaml:568]: oplogSpanMin:  
we need to add the value

**Comment by hors** [comment]
I think we need to have a test for all these options.

**Comment by inelpandzic** [comment]
> I think we need to have a test for all these options.

You're right, I'll add them.

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| arbiter | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/arbiter.log) |
| balancer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/balancer.log) |
| custom-replset-name | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/custom-replset-name.log) |
| cross-site-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/cross-site-sharded.log) |
| data-at-rest-encryption | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/data-at-rest-encryption.log) |
| data-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/data-sharded.log) |
| demand-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/demand-backup.log) |
| demand-backup-eks-credentials | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/demand-backup-eks-credentials.log) |
| demand-backup-physical | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/demand-backup-physical.log) |
| demand-backup-physical-sharded | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/demand-backup-physical-sharded.log) |
| demand-backup-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/demand-backup-sharded.log) |
| expose-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/expose-sharded.log) |
| ignore-labels-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/ignore-labels-annotations.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/init-deploy.log) |
| finalizer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/finalizer.log) |
| ldap | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/ldap.log) |
| ldap-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/ldap-tls.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/limits.log) |
| liveness | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/liveness.log) |
| mongod-major-upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/mongod-major-upgrade.log) |
| mongod-major-upgrade-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/mongod-major-upgrade-sharded.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/monitoring-2-0.log) |
| multi-cluster-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/multi-cluster-service.log) |
| non-voting | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/non-voting.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/one-pod.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/operator-self-healing-chaos.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/pitr.log) |
| pitr-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/pitr-sharded.log) |
| pitr-physical | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/pitr-physical.log) |
| recover-no-primary | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/recover-no-primary.log) |
| rs-shard-migration | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/rs-shard-migration.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/scheduled-backup.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/security-context.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/self-healing-chaos.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/service-per-pod.log) |
| serviceless-external-nodes | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/serviceless-external-nodes.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/smart-update.log) |
| split-horizon | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/split-horizon.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/storage.log) |
| tls-issue-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/tls-issue-cert-manager.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/upgrade.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/upgrade-consistency.log) |
| upgrade-consistency-sharded-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/upgrade-consistency-sharded-tls.log) |
| upgrade-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/upgrade-sharded.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/b86f3f41/version-service.log) |
| We run 47 out of 47|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/1517/commits/b86f3f41de2ee04cc9236f81cb05cb9d338e3f41
image: `perconalab/percona-server-mongodb-operator:PR-1517-b86f3f41`

**Review by nmarukovich — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| arbiter | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/arbiter.log) |
| balancer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/balancer.log) |
| custom-replset-name | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/custom-replset-name.log) |
| cross-site-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/cross-site-sharded.log) |
| data-at-rest-encryption | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/data-at-rest-encryption.log) |
| data-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/data-sharded.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/demand-backup.log) |
| demand-backup-eks-credentials | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/demand-backup-eks-credentials.log) |
| demand-backup-physical | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/demand-backup-physical.log) |
| demand-backup-physical-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/demand-backup-physical-sharded.log) |
| demand-backup-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/demand-backup-sharded.log) |
| expose-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/expose-sharded.log) |
| ignore-labels-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/ignore-labels-annotations.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/init-deploy.log) |
| finalizer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/finalizer.log) |
| ldap | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/ldap.log) |
| ldap-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/ldap-tls.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/limits.log) |
| liveness | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/liveness.log) |
| mongod-major-upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/mongod-major-upgrade.log) |
| mongod-major-upgrade-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/mongod-major-upgrade-sharded.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/monitoring-2-0.log) |
| multi-cluster-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/multi-cluster-service.log) |
| non-voting | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/non-voting.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/one-pod.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/operator-self-healing-chaos.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/pitr.log) |
| pitr-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/pitr-sharded.log) |
| pitr-physical | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/pitr-physical.log) |
| recover-no-primary | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/recover-no-primary.log) |
| rs-shard-migration | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/rs-shard-migration.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/scheduled-backup.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/security-context.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/self-healing-chaos.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/service-per-pod.log) |
| serviceless-external-nodes | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/serviceless-external-nodes.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/smart-update.log) |
| split-horizon | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/split-horizon.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/storage.log) |
| tls-issue-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/tls-issue-cert-manager.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/upgrade.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/upgrade-consistency.log) |
| upgrade-consistency-sharded-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/upgrade-consistency-sharded-tls.log) |
| upgrade-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/upgrade-sharded.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/439e8c67/version-service.log) |
| We run 47 out of 47|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/1517/commits/439e8c673b47122aa631a3fcfedac96a4612d707
image: `perconalab/percona-server-mongodb-operator:PR-1517-439e8c67`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| arbiter | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/arbiter.log) |
| balancer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/balancer.log) |
| custom-replset-name | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/custom-replset-name.log) |
| cross-site-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/cross-site-sharded.log) |
| data-at-rest-encryption | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/data-at-rest-encryption.log) |
| data-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/data-sharded.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/demand-backup.log) |
| demand-backup-eks-credentials | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/demand-backup-eks-credentials.log) |
| demand-backup-physical | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/demand-backup-physical.log) |
| demand-backup-physical-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/demand-backup-physical-sharded.log) |
| demand-backup-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/demand-backup-sharded.log) |
| expose-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/expose-sharded.log) |
| ignore-labels-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/ignore-labels-annotations.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/init-deploy.log) |
| finalizer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/finalizer.log) |
| ldap | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/ldap.log) |
| ldap-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/ldap-tls.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/limits.log) |
| liveness | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/liveness.log) |
| mongod-major-upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/mongod-major-upgrade.log) |
| mongod-major-upgrade-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/mongod-major-upgrade-sharded.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/monitoring-2-0.log) |
| multi-cluster-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/multi-cluster-service.log) |
| non-voting | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/non-voting.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/one-pod.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/operator-self-healing-chaos.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/pitr.log) |
| pitr-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/pitr-sharded.log) |
| pitr-physical | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/pitr-physical.log) |
| pvc-resize | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/pvc-resize.log) |
| recover-no-primary | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/recover-no-primary.log) |
| rs-shard-migration | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/rs-shard-migration.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/scheduled-backup.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/security-context.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/self-healing-chaos.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/service-per-pod.log) |
| serviceless-external-nodes | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/serviceless-external-nodes.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/smart-update.log) |
| split-horizon | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/split-horizon.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/storage.log) |
| tls-issue-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/tls-issue-cert-manager.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/upgrade.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/upgrade-consistency.log) |
| upgrade-consistency-sharded-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/upgrade-consistency-sharded-tls.log) |
| upgrade-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/upgrade-sharded.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1517/0a47abe1/version-service.log) |
| We run 48 out of 48|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/1517/commits/0a47abe1ec19571058c9082cad3235cfdafda018
image: `perconalab/percona-server-mongodb-operator:PR-1517-0a47abe1`

**Review by nmarukovich — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**